python/ml4ir/base/io/local_io.py

```diff
@@ -147,6 +147,8 @@ def write_df(self, df, outfile: str = None, sep: str = ",", index: bool = True)
             dataframe in csv form if outfile is None
         """
         self.log("Writing dataframe to : {}".format(outfile))
+        np.set_printoptions(formatter={'all': lambda x: str(x.decode('utf-8')) if isinstance(x, bytes) else str(x)},
```

Comment:
Good one. Worth leaving a note why you are doing this. I mean, I know, but good for documentation.
@ducouloa